### ISSUE TITLE
How to assign an link (onclik) and picture in Grid?

### ISSUE THREAD

**Issue by saamy91** [GavickPro/Grid-GK5#22]
Hello,

As in my request title, how can i assign an picture with an link to a grid position for Metro Grid Module (joomla 3) ?

I search in your documentation and FAQ but i don't find anythink about it.

Advance thanks.

**Comment by dziudek** [comment]
Hello,

In this case you have to assign in the grid module position a custom HTML module with i.e. the following code:

``` html
<a href="YOUR_URL" style="background-image: url('YOUR_IMAGE_PATH');"></a>
```

And of course please use the following CSS code:

``` css
.gkGridElement a {
background-size: cover;
background-position: center center;
background-repeat: no-repeat;
height: 100%;
left: 0;
position: absolute;
top: 0;
width: 100%;
}
```

**Comment by saamy91** [comment]
Hello,

Thank your for your reply.
I don't understand your explanation to be frank.
I need to create an custom HTML Module, then choose grid module position ?
But i don't have anythink about grid module in position :

[image: Images intégrées 1]

2015-04-29 16:06 GMT+02:00 Tomasz Dziuda notifications@github.com:

**Comment by dziudek** [comment]
Please read: https://github.com/GavickPro/Grid-GK5/issues/21
